dev/util.js

```diff
@@ -57,7 +57,7 @@ export function getAllFiles(baseDirectory, predicate = null) {
 export function createDictionaryArchive(dictionaryDirectory, dictionaryName) {
     const fileNames = fs.readdirSync(dictionaryDirectory);
 
-    // const zipFileWriter = new BlobWriter();
+    // Const zipFileWriter = new BlobWriter();
```

Comment:
Hmm, I guess we should just eliminate this commented out code?

Reply:
I'm trying to address this now in a separate PR, it was easiest to just allow it to auto-fix this code and it will be removed soon after.